src/game/GameResults.as

```diff
@@ -205,6 +208,12 @@ package game
             navSaveReplay.addEventListener(MouseEvent.CLICK, eventHandler);
             this.addChild(navSaveReplay);
 
+            navRandomSong = new BoxButton(110, 32, "Play Random Song");
```

Comment:
Can you change the text `"Play Random Song"` into `_lang.string("game_results_play_random_song")` so that translations can be applied?